src/form/dutch-form-fields.test.ts

```diff
@@ -0,0 +1,343 @@
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('buitenlands adres - regel 2', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'Adres, regel 2',
+          fieldType: 'input',
+          inputSubtype: 'text',
+        },
+        definition: {
+          ...adresRegel2Validation,
+        },
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('buitenlands adres - regel 3', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'Adres, regel 3',
+          fieldType: 'input',
+          inputSubtype: 'text',
+        },
+        definition: {
+          ...adresRegel3Validation,
+        },
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('e-mailadres', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'E-mailadres',
+          fieldType: 'input',
+          inputSubtype: 'email',
+        },
+        definition: {
+          autoComplete: 'email',
+        },
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('burgerservicenummer (BSN)', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'Burgerservicenummer',
+          fieldType: 'input',
+          inputSubtype: 'text',
+        },
+        definition: {
+          ...bsnValidation,
+        },
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('geboorteplaats', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'Geboorteplaats',
+          fieldType: 'input',
+          inputSubtype: 'text',
+        },
+        definition: {
+          ...placeOfBirthValidation,
+        },
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('nummer reisdocument', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'Nummer Nederlands reisdocument',
+          fieldType: 'input',
+          inputSubtype: 'text',
+        },
+        definition: {
+          ...reisdocumentValidation,
+        },
+      });
+      expect(def).not.toBeUndefined();
+    });
+  });
+
+  describe('KVK-nummer', () => {
+    it('should support its definition', () => {
+      const def = createFormField({
+        declaration: {
+          id,
+          label: 'KVK-nummer',
+          fieldType: 'input',
+          inputSubtype: 'text',
+        },
+        definition: {
+          ...kvkValidation,
+        },
+      });
+      expect(def).not.toBeUndefined();
```

Comment:
Hmmm should this maybe be `const {definition} = createFormField({...}); expect(definition).not.toBeUndefined()`?

Reply:
These tests are not really worked out to test anything specific, they mostly test they don't throw an error.